### PR TITLE
Added docs to fix CORS errors when running development server

### PR DESCRIPTION
**Pull request by emmanueletukudo**
### Scope & Purpose

This PR adds documentation on how to fix cross-origin-policy (CORS) errors on a development server.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -621,28 +649,28 @@ The symbolserver over at https://download.arangodb.com/symsrv_arangodbXX/ is
...
-    - arangobackup
-    - arangobench
-    - arangodump
-    - arangoexport
-    - arangoimport
-    - arangorestore
-    - arangosh
-    - arangovpack
+- arango - the basic arangodb library needed by all components
+- arango_v8 - the basic V8 wrappers needed by all components
```
Comment (dothebart) [inline CONTRIBUTING.md:653]: make it lib/v8 pls.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -621,28 +649,28 @@ The symbolserver over at https://download.arangodb.com/symsrv_arangodbXX/ is
...
-  - the client utilities:
-    - arangobackup
-    - arangobench
-    - arangodump
-    - arangoexport
-    - arangoimport
-    - arangorestore
-    - arangosh
-    - arangovpack
+- arango - the basic arangodb library needed by all components
```
Comment (dothebart) [inline CONTRIBUTING.md:652]: make it lib/ pls.

**Review by dothebart — APPROVED** [review]
please also correct these two directories.

File: CONTRIBUTING.md (hunk 1 above)
Comment (emmanueletukudo) [reply CONTRIBUTING.md:653]: @dothebart do you mean `arango/v8` instead of `arango_v8`?

Comment (dothebart) [reply CONTRIBUTING.md:653]: we don't have an arango/ directory anymore, its lib/ nowadays. and below it there's V8

File: CONTRIBUTING.md (hunk 2 above)
Comment (emmanueletukudo) [reply CONTRIBUTING.md]: Hello @dothebart, I just pushed an update to fix this.

**Review by dothebart — APPROVED** [review]

**Comment by Zyqsempai** [comment]
@emmanueletukudo Can we move it forward? First of all by fixing comments.

**Pull request merged**